pre_commit_hooks/file_contents_sorter.py

```diff
@@ -18,19 +18,18 @@
 
 
 def sort_file_contents(f):
-    before = tuple(f)
+    before = [line.strip(b'\n\r') for line in f if line.strip()]
```

Comment:
Yes it will, good catch!! Hmm if it rewrites file it should remove blank lines (otherwise they'd just end up all sorted together which is useless) but it shouldn't rewrite the file just to remove blank lines. I'll add test case for this with the fix.